circle.yml

```diff
@@ -20,3 +20,4 @@ deployment:
     branch: master
     commands:
       - NODE_ENV=prod npm run build
+      - npm publish
```

Comment:
@donnapep Okay got it. What does `prepublish` script in package.json get used for?